### DISCUSSION TITLE
repeat command without text on screen, run command in background?

### DISCUSSION THREAD

**Discussion by NeedNot** [gopro/labs#406] [Q&A]
i read through the different pages of docs and i feel like i got the hang of it pretty easily and i quickly wrote this script for the max

<m1!B1+=A+1~=A0>A5<A7oC0+oV9+oW0!5R

which is basically this
```python
A = 0
while True:
    if detect_motion(0.01):
        beep_once()
        A += 1
    else:
        A = 0
    if A > 5:
        if A < 7:
            disable_auto_shut_off()
            set_beep_volume_max()
            disable_wifi()       
    time.sleep(5)
```

the idea is if you unfortunatly lose your gopro in water it should give you a chance to help find it.

it works well and can even run at the same time as recording but i have a few questions

can i disable the text on screen, it's very annoying to see the count down the entire time.
can i make the qr code not stop when a button is pressed, currently you must start recording using your voice so you dont stop it. it would also be nice for it to run until power off that way you never have to worry about losing it when you weren't expecting it.
why doesn't it work when not in 360 mode? I'm on the max and if the camera is in 360 photo mode, video mode, or timelapes it works but if it's set to hero mode it doesn't work, i only use my max for 360 so it shouldn't be too much of an issue but always nice to have.

for other models:
from what i understand only max and hero 11 have the scripting support to this level? so how could this be achived on older cameras?
in my last post you mentioned hero 11 had a light sensor? i cannot seem to find anything about that but it would be a nice feature to for hero 11 owners.

**Comment by dnewman-gpsw** [comment]
Try this:  <m1!B1+=A+1~=A0>A5<A7oC0+oV9+oW0!5N!R

Using !5N <- Sleep(5) doesn't display anything.

It worked in HERO mode when I tried.

HERO10 might get these feature, but only if it gets another  mainline firmware update (Labs generally follows those.)  I don't expect any more updates for older cameras.

The image sensor is the only light sensor on GoPro cameras.
